tests/unit/src/main/scala/tests/BaseLspSuite.scala

```diff
@@ -81,6 +78,18 @@ abstract class BaseLspSuite(suiteName: String) extends BaseSuite {
     server.server.userConfig = this.userConfig
   }
 
+  def cancelServer(): Unit = {
+    if (server != null) {
+      server.server.cancel()
+    }
+  }
+
+  override def beforeEach(context: BeforeEach): Unit = {
+    cancelServer()
+    if (context.test.tags.contains(Ignore)) return
+    newServer(context.test.name)
+  }
```

Comment:
moved `cancelServer` and `newServer` to separate methods to that we can invoke them without having to invoke `beforeEach`